package.json

```diff
@@ -40,8 +40,11 @@
   "dependencies": {
     "@babel/polyfill": "^7.7.0",
     "bignumber.js": "^8.1.1",
+    "bip32": "^2.0.5",
     "bitcoin-address-validation": "^0.2.9",
     "bitcoinjs-lib": "^4.0.5",
+    "bs58check": "^2.1.2",
+    "bufio": "^1.0.6",
```

Comment:
This is for using a JS `Struct` object as well as Buffer reader/writer. (from the bcoin-org project)